### DISCUSSION TITLE
It seems that the conditions for achieving Galaxy Brain are very confusing

### DISCUSSION THREAD

**Discussion by qhy040404** [Schweinepriester/github-profile-achievements#17] [Q&A] [answered]
I tested many times in my test repo, including private and public. And they don't seem to work. Then I opened another test repository in my bot account, but the achievement is still not achieved. 
So confusing. 😵‍💫 

Addition to answer: 
Make sure you haven't marked your own answer in your own question.

**Comment by le2sky** [comment, marked as answer]
hello! @qhy040404 😀 
if there is more than two accepted answer in the discussion, it's sure to get you a galaxy brain badge.
I check your situation and googled related questions. and i found  some checklists to get a galaxy brain.

to achieve the 'Galaxy Brain' achievement, take note of the following:
- Answer at least 2 discussions.
- Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
- Discussions should be in a public repository.
- There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

I hope this helps you to get the Galaxy Brain by referring to it.  have a nice day🖐
https://github.com/github-community/community/discussions/18293


There is also a good explanation here.
https://github.com/Schweinepriester/github-profile-achievements/issues/15

**Reply by qhy040404** [reply]
Thanks for your help. 
My situation seems to be similar to the last one based on the advice you provided. 
I'll try to unmark my answers that is marked by myself. And check if the achievement is achieved.

**Reply by le2sky** [reply]
@qhy040404 

Ironically, I have one answer left to accept to receive the badge. If you accept my answer
 I will receive a galaxy brain. haha😂

**Reply by qhy040404** [reply]
> There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

@le2sky 

I have a question about this one. 
Do I need to answer another discussion to get after I unmark invalid answers?

**Reply by qhy040404** [reply]
> > There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.
> 
> @le2sky
> 
> I have a question about this one. Do I need to answer another discussion to get after I unmark invalid answers?

I tested in qhyBot/GBtest#5 . It works. Thanks! 👍🏻 ❤️ 

And congratulations 🎉

**Reply by le2sky** [reply]
It is a nobel idea. if that happens, my galaxy brain is your haha 
I'm glad it was helpful. We are now the Galaxy Brain 👍👍👍

**Comment by ItzJustNick** [comment]
I think you need to answer 2 disscusions and then become the accepted answer on both to get the achivement. Hope this helps.
